src/FSharp.Control.TaskSeq/TaskSeq.fs

```diff
@@ -15,6 +15,20 @@ module TaskSeq =
         for c: 'T in [] do
             yield c
     }
+    
+    let singleton (source: 'T) =
```

Comment:
Please move it to `TaskSeqInternal` and call that function from here.

Reply:
Not clear to me this request.
The idea is that `singleton` is a public function.
Why do you want to move it to an internal namespace?
Is it to re-use it within other internal functions?

Reply:
Yes. It’s nothing more than organisational. Also, we group like functions. Say we add `singletonAsync`, we’d process both in a single function in `TaskSeqInternal`.

This division is slightly arbitrary, but I like to keep the larger details separate from the lean and mean implementation file. In the end, there’s no functional reason other than organisation and style.

Reply:
Btw, just do be clear, the public function remains in `TaskSeq.fs`. That file should contain all surface area public functions. Only the implementation moves. Sorry for being vague earlier.

Reply:
This is now done. I've also fixed the `singleton` implementation to properly return `defaultof` whenever before/after iteration.